src/components/TxData.js

```diff
@@ -61,6 +63,14 @@ class TxData extends React.Component {
 
   componentDidUpdate = (prevProps) => {
     if (prevProps.transaction !== this.props.transaction) {
+      // When we are changing the transaction but keeping the same component
+      // we must clean the state
+      this.tokensFound = [];
+      if (this.state.raw) {
+        // XXX Show raw transaction should come directly from state, we should refactor this later
+        this.toggleRaw();
```

Comment:
Should we call it after reseting the state? I feel we might have a minor bug here.

Reply:
This code is outdated